Delves/TheSinkhole/Locales/frFR.lua

```diff
@@ -8,5 +8,5 @@ L = BigWigs:NewBossLocale("Leviathan Caller", "frFR")
 if L then
 	L.leviathan_caller = "Cor du léviathan"
 	L.guardian_tentacle = "Tentacule gardien"
-	--L.slamming_tentacles = "Slamming Tentacles" -- plural
```

Comment:
Is this right? It's a different NPC than the above, wowhead: https://www.wowhead.com/fr/npc=220741/tentacule-percutant